### ISSUE TITLE
Adding auth via the Sln builder generates incorrect code

### ISSUE THREAD

**Issue by RhysC** [Particular/ServiceMatrix#444]
(NSB core 5.2.0, Service matrix 2.2.2)
Seems to be a bug in this file:
ServiceMatrix/src/ServiceMatrix/T/T4/Security/EndpointAuthorizeOutgoingMessages.v5.0.tt

The method signature does not match with the Interface specified

sample of my generated file:  
```CSharp  

using System;
using NServiceBus;
using NServiceBus.MessageMutator;

namespace Matcha.Web.Infrastructure
{
    public partial class AuthorizeOutgoingMessages : Matcha.Infrastructure.Security.AuthorizeOutgoingMessages, IMutateOutgoingTransportMessages
    {
    }

    public class RegisterMutator : INeedInitialization
    {
        public void Init(Configure config)
        {
            config.Configurer.ConfigureComponent<AuthorizeOutgoingMessages>(DependencyLifecycle.SingleInstance);
        }
    }
}  

```
expected implementation:  

```CSharp  
        public void Customize(BusConfiguration configuration)//NOT void Init(Configure config)
        {
            throw new NotImplementedException();
        }
```

**Comment by RhysC** [comment]
I assume the change required is: 
```csharp
 public void Customize(BusConfiguration configuration)
        {
            configuration.RegisterComponents(c => c.ConfigureComponent<AuthorizeOutgoingMessages>(DependencyLifecycle.InstancePerCall));
        }
```

**Comment by RhysC** [comment]
does anyone look at these... this is a trivial change that breaks....on every..... single..... build....

**Comment by indualagarsamy** [comment]
hey @RhysC - Sorry, I missed this. Let me get back to you on this.

**Comment by indualagarsamy** [comment]
@RhysC - I'll get this out as soon as I've tested this change. Thanks for your input and adding more information regarding the actual fix involved.

**Comment by indualagarsamy** [comment]
Fixed in 2.2.3

**Comment by indualagarsamy** [comment]
hey @RhysC - v2.2.3 is now out. Thanks for your time.
